grunt/aliases.yaml

```diff
@@ -2,9 +2,9 @@ development:
     description: 'Development build task'
     tasks:
         - 'clean:dist'
-        - 'sass:development'
-        - 'postcss:autoprefixer'
         - 'lint'
+        - 'sass:development'
+        - 'postcss:cssDevelopment'
```

Comment:
Maybe we can combine the `sass` and the `postcss` task to one `styles` task and call it combined, so we do not have to call the whole `development` task when we just want to build the css.

Reply:
Should we call this task stylesDev to recognize if it's building the development or the production CSS?

Reply:
Please add one task for production as well.